packages/components/src/components/side-note/side-note.tsx

```diff
@@ -4,21 +4,32 @@ import Icon from '../icon/icon';
 import Text from '../text';
 import './side-note.scss';
 
+const type_to_icon_mapper = {
```

Comment:
It's just a suggestion:


```suggestion
const type_icon_mapper = {
```